x/group/keeper/keeper_test.go

```diff
@@ -0,0 +1,2143 @@
+package keeper_test
+
+import (
+	"bytes"
+	"context"
+	"sort"
+	"strings"
+	"time"
+
+	gogotypes "github.com/gogo/protobuf/types"
+	"github.com/stretchr/testify/suite"
+
+	"github.com/cosmos/cosmos-sdk/types"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	servermodule "github.com/cosmos/cosmos-sdk/types/module/server"
+	"github.com/cosmos/cosmos-sdk/types/testutil"
+	authkeeper "github.com/cosmos/cosmos-sdk/x/auth/keeper"
+	bankkeeper "github.com/cosmos/cosmos-sdk/x/bank/keeper"
+	banktypes "github.com/cosmos/cosmos-sdk/x/bank/types"
+	"github.com/cosmos/cosmos-sdk/x/group"
+	mintkeeper "github.com/cosmos/cosmos-sdk/x/mint/keeper"
+	minttypes "github.com/cosmos/cosmos-sdk/x/mint/types"
+	paramstypes "github.com/cosmos/cosmos-sdk/x/params/types"
+)
+
+type KeeperTestSuite struct {
```

Comment:
let's just use the same kind of test suite as for other keepers in cosmos-sdk (based on simapp for now)